src/category_theory/limits/concrete_category.lean

```diff
@@ -24,30 +25,80 @@ local attribute [instance] concrete_category.has_coe_to_fun
 -- We now prove a lemma about naturality of cones over functors into bundled categories.
 namespace cone
 
-variables {J : Type u} [small_category J]
-
 /-- Naturality of a cone over functors to a concrete category. -/
-@[simp] lemma naturality_concrete {G : J ⥤ C} (s : cone G) {j j' : J} (f : j ⟶ j') (x : s.X) :
-   (G.map f) ((s.π.app j) x) = (s.π.app j') x :=
+@[simp] lemma w_apply {F : J ⥤ C} (s : cone F) {j j' : J} (f : j ⟶ j') (x : s.X) :
+   F.map f (s.π.app j x) = s.π.app j' x :=
 begin
-  convert congr_fun (congr_arg (λ k : s.X ⟶ G.obj j', (k : s.X → G.obj j')) (s.π.naturality f).symm) x;
-  { dsimp, simp [-cone.w] },
+  convert congr_fun (congr_arg (λ k : s.X ⟶ F.obj j', (k : s.X → F.obj j')) (s.w f)) x,
+  simp only [coe_comp],
 end
 
+@[simp]
+lemma w_forget_apply (F : J ⥤ C) (s : cone (F ⋙ forget C)) {j j' : J} (f : j ⟶ j') (x : s.X) :
+  F.map f (s.π.app j x) = s.π.app j' x :=
+congr_fun (s.w f : _) x
+
 end cone
 
 namespace cocone
 
-variables {J : Type u} [small_category J]
-
 /-- Naturality of a cocone over functors into a concrete category. -/
-@[simp] lemma naturality_concrete {G : J ⥤ C} (s : cocone G) {j j' : J} (f : j ⟶ j') (x : G.obj j) :
-  (s.ι.app j') ((G.map f) x) = (s.ι.app j) x :=
+@[simp] lemma w_apply {F : J ⥤ C} (s : cocone F) {j j' : J} (f : j ⟶ j') (x : F.obj j) :
+  s.ι.app j' (F.map f x) = s.ι.app j x :=
 begin
-  convert congr_fun (congr_arg (λ k : G.obj j ⟶ s.X, (k : G.obj j → s.X)) (s.ι.naturality f)) x;
-  { dsimp, simp [-nat_trans.naturality, -cocone.w] },
+  convert congr_fun (congr_arg (λ k : F.obj j ⟶ s.X, (k : F.obj j → s.X)) (s.w f)) x,
+  simp only [coe_comp],
 end
 
+@[simp]
+lemma w_forget_apply (F : J ⥤ C) (s : cocone (F ⋙ forget C)) {j j' : J} (f : j ⟶ j') (x : F.obj j) :
```

Comment:
Why this name? There is no `w` in the statement.

Reply:
It continues a not-very-helpful naming convention, starting in `cone`, where the fact that the legs of a cone form a commutative triangle with the morphisms provided by the functor is called `cone.w`. We then have `limit.w`, which is this condition for a limit cone, etc.

Since this lemma is just an application of the fact `s.w` to an element `x`, I put `w` in the name.

A reasonable suggestion would be the rename this (back on `cone`, and everywhere thereafter) as `ι_app_map`. I can do that in a separate PR (although I'm a little hesitant making widespread changes across the `limits` directory while `prop_limits` is still in play).

Reply:
Ok, fair enough